src/containers/pages/FocusInvestigation/single/index.tsx

```diff
@@ -169,6 +171,9 @@ class SingleFI extends React.Component<RouteComponentProps<RouteParams> & Single
 
     return (
       <div className="mb-5">
+        <Helmet>
+          <title>{FI_SINGLE_TITLE}</title>
```

Comment:
This title should be the name of the focus area, something like: `Focus Investigations in Tha Sen 8 |  Reveal`